Sofascore2025/Sofascore2025/ViewController.swift

```diff
@@ -1,19 +1,64 @@
-//
-//  ViewController.swift
-//  Sofascore2025
-//
-//  Created by Niko on 06.03.2025..
-//
-
 import UIKit
+import SnapKit
+import SofaAcademic
 
-class ViewController: UIViewController {
-
+class ViewController: UIViewController, BaseViewProtocol {
+    
+    private let dataSource = Homework2DataSource()
+    private var events: [Event] = []
+    private let leagueView: LeagueView
```

Comment:
ti bi tu trebao imat ovako nesto 
```suggestion
    private let leagueView: LeagueView = .init()
```